mpolyakov/homework_4/olx_scraping/output.json

```diff
@@ -0,0 +1,10 @@
+{"name": "Батарея аккумулятор 53 5560 5740 42 Acer Aspire 4253 5250 53 5551 52", "url": "https://www.olx.ua/obyavlenie/batareya-akkumulyator-53-5560-5740-42-acer-aspire-4253-5250-53-5551-52-IDHg88i.html", "price": 477, "currency": " грн.", "category": "Новый", "state": "Новый", "description": "Батарея аккумулятор  53 5560 5740 42 Acer Aspire 4253 5250 53 5551 52", "date_of_creation": "в 23:11, 4 октября 2020", "views_count": "979", "id": "639226962", "photos_urls": "https://ireland.apollo.olxcdn.com:443/v1/files/tfptctpfl8yb-UA/image;s=1000x700", "seller_name": "Ирина", "seller_address": null, "olx_delivery_availability": "Купить с  доставкой"}
+{"name": "GERMANY! Пылесос Kassel 5078B, колбовый, 3L- 3000W. Циклонного типа", "url": "https://www.olx.ua/obyavlenie/germany-pylesos-kassel-5078b-kolbovyy-3l-3000w-tsiklonnogo-tipa-IDG5yvA.html", "price": 1195, "currency": "  грн.", "category": "Пылесосы", "state": "Новый", "description": "GERMANY! Пылесос Kassel 5078B, колбовый, 3L- 3000W. Циклонного типа без мешка в магазине шоурум \"POLZ\"", "date_of_creation": "в 09:14, 23 октября 2020", "views_count": "5557", "id": "621930468", "photos_urls": "https://ireland.apollo.olxcdn.com:443/v1/files/be69n9joeu761-UA/image;s=1000x700", "seller_name": "Polz.com.ua", "seller_address": "Киев, Киевская область, Оболонский", "olx_delivery_availability": "Купить с  доставкой"}
+{"name": "Компьютер для учебы, работы, отдыха", "url": "https://www.olx.ua/obyavlenie/kompyuter-dlya-ucheby-raboty-otdyha-IDJ3Mpb.html", "price": 5500, "currency": "  грн.", "category": "Настольные компьютеры", "state": "Новый", "description": "Прoцеccор АMD Athlon 200GE этo пpоизводительноe peшeние для дoмашнeй станции или paбoчего ПK. Эта мoдeль имeeт 2 ядра/4 пoтока c бaзовoй тaктoвoй частотой 3200 МГц и oснaщаeтcя встpоенным видeoядpом Radeon Vеga 3 с видеoпамятью 2Гб(выдeляeтся из оперативной).", "date_of_creation": "в 09:14, 23 октября 2020", "views_count": "947", "id": "665836177", "photos_urls": "https://ireland.apollo.olxcdn.com:443/v1/files/nfbor2pktf6a2-UA/image;s=1000x700", "seller_name": "Сергей", "seller_address": "Запорожье, Запорожская область, Шевченковский", "olx_delivery_availability": "Купить с  доставкой"}
+{"name": "IQOS 3 duos", "url": "https://www.olx.ua/obyavlenie/iqos-3-duos-IDJAYhX.html", "price": 599, "currency": " грн.", "category": "Б/у", "state": "Б/у", "description": "Айкос 3 дуос", "date_of_creation": "в 09:14, 23 октября 2020", "views_count": "9", "id": "673746681", "photos_urls": "https://ireland.apollo.olxcdn.com:443/v1/files/3yzpcgn56j9h1-UA/image;s=1000x700", "seller_name": "Анатолий", "seller_address": "Киев, Киевская область, Деснянский", "olx_delivery_availability": ""}
```

Comment:
category and state are the same? 🤔

Reply:
I'm sure that photos_urls should contain array of urls

Reply:
views_count should be int, I guess

Reply:
what will happen with the price in case of USD price?

Reply:
It works: "currency": " $",